public/index.html

```diff
@@ -29,24 +29,21 @@
     <meta property="og:description" content="Australian Government dashboard to measure the performance of Australia's largest cities against the National Cities Performance Framework.">
     <meta property="og:title" content="Smart Cities Performance Dashboard">
     <meta property="og:url" content="https://smart-cities.dashboard.gov.au">
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+      gtag('config', 'UA-61222473-12');
+    </script>
```

Comment:
Moved this up here to ensure `gtag` exists regardless of where the react script gets injected below.